src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/rds/CertificateBuilder.kt

```diff
@@ -101,26 +101,35 @@ fun printRequestStatusBuilder(
 }
 
 fun rdsElectoralRegistrationOfficeBuilder(
-    name: String = aValidEroName()
+    name: String = aValidEroName(),
+    address: Address = rdsAddressBuilder()
 ): ElectoralRegistrationOffice {
     return ElectoralRegistrationOffice(
-        address = Address(
-            street = aValidAddressStreet(),
-            postcode = aValidAddressPostcode()
-        ),
+        address = address,
         name = name,
         phoneNumber = aValidPhoneNumber(),
         emailAddress = aValidEmailAddress(),
         website = aValidWebsite()
     )
 }
 
-fun rdsAddressBuilder(): Address {
-    return Address(
-        street = aValidAddressStreet(),
-        postcode = aValidAddressPostcode()
-    )
-}
+fun rdsAddressBuilder(
+    street: String = DataFaker.faker.address().streetName(),
```

Comment:
Now using this one in other tests it needs to be fully populated.

RDS prefix will be removed and packaging restructured in separate PR.